src/Components/Lists/EquipmentDetailList/EquipmentDetailList.css

```diff
@@ -0,0 +1,36 @@
+.EquipmentDetailList-Container {
```

Comment:
Using comments to explain a bit of CSS, even though it will become a bit redundant, will make everything clear and clean :)

Reply:
Addressed!